pkg/odo/cli/version/version.go

```diff
@@ -23,58 +23,100 @@ var (
 	GITCOMMIT = "HEAD"
 )
 
-var clientFlag bool
-
-// versionCmd represents the version command
-var versionCmd = &cobra.Command{
-	Use:   "version",
-	Short: "Print the client version information",
-	Long:  "Print the client version information",
-	Example: `  # Print the client version of Odo
-  odo version
-	`,
-	Run: func(cmd *cobra.Command, args []string) {
-
-		// If verbose mode is enabled, dump all KUBECLT_* env variables
-		// this is usefull for debuging oc plugin integration
-		for _, v := range os.Environ() {
-			if strings.HasPrefix(v, "KUBECTL_") {
-				glog.V(4).Info(v)
-			}
+// RecommendedCommandName is the recommended version command name
+const RecommendedCommandName = "version"
+
+var versionLongDesc = ktemplates.LongDesc("Print the client version information")
+
+var versionExample = ktemplates.Examples(`
+# Print the client version of Odo
+%[1]s`,
+)
+
+// Options encapsulates all options for odo version command
+type Options struct {
+	// clientFlag indicates if the user only wants client information
+	clientFlag bool
+	// generic context options common to all commands
+	*genericclioptions.Context
```

Comment:
This is unneeded for this command, actually. Wondering if I should remove it from some of the `catalog` commands as well, actually :)

Reply:
Removed